staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode.go

```diff
@@ -96,6 +100,12 @@ var Decode cbor.DecMode = func() cbor.DecMode {
 		// representation (RFC 8259 Section 6).
 		NaN: cbor.NaNDecodeForbidden,
 		Inf: cbor.InfDecodeForbidden,
+
+		// When unmarshaling a byte string into a []byte, assume that the byte string
+		// contains base64-encoded bytes, unless explicitly counterindicated by an "expected
+		// later encoding" tag. This is consistent with the because of unmarshaling a JSON
+		// text into a []byte.
+		ByteStringExpectedFormat: cbor.ByteStringExpectedBase64,
```

Comment:
does this use a base64 encoding consistent with json (base64 StdEncoding, rather than URLEncoding or RawStdEncoding)?

Reply:
It does, there is a distinct tag for base64url:

https://github.com/kubernetes/kubernetes/blob/8c508c5480844682eab8a4172526b6db3476fdc0/vendor/github.com/fxamacker/cbor/v2/decode.go#L2177
https://github.com/kubernetes/kubernetes/blob/master/vendor/sigs.k8s.io/json/internal/golang/encoding/json/decode.go#L1059